src/cluster/cluster-registry.js

```diff
@@ -61,6 +61,11 @@ module.exports = class ClusterRegistry extends EventEmitter{
 			action: C.ACTIONS.REMOVE,
 			data:[ this._options.serverName ]
 		});
+
+		// TODO: If a message connector doesn't close this is required to avoid an error
+		// being thrown during shutdown
+		this._options.messageConnector.subscribe( C.TOPIC.CLUSTER, () => {} );
```

Comment:
This is because message connectors don't have a close function, meaning it still recieves messages after it is removed.